app/models/work-item.ts

```diff
@@ -112,14 +112,14 @@ export async function getNextWorkItem(
 ): Promise<WorkItem> {
   let workItemData;
   const acceptableJobStatuses = _.cloneDeep(activeJobStatuses);
-  // The query-cmr service should keep going for paused jobs to avoid the ten minute CMR
-  // scroll session timeout
+  // Now that we use search-after instead of scrolling we could allow pausing of query-cmr
```

Comment:
This comment is slightly confusing since you didn't make that change. Maybe add a TODO?